Preprocessing breaks if integration folder is not below package root

If the integration folder containing `.feature` files is not below the package root where `cypress-cucumber-preprocessor` is installed, preprocessing breaks due to unresolvable dependencies:
```
Error: Cannot find module 'cypress-cucumber-preprocessor/steps' from '[redacted]/cucumber/Cucumber'


This occurred while Cypress was compiling and bundling your test code. This is usually caused by:

- A missing file or dependency
- A syntax error in the file or one of its dependencies
```

Example repository: https://github.com/pmelab/cypress-cucumber-preprocessor-example

I think this could be solved by injecting requirements as globals instead of requiring them from inside the template? I don't know enough about the plugin system and browserify, but I would be happy to have a look if I get some pointers.

--- a/lib/cucumberTemplate.js
+++ b/lib/cucumberTemplate.js
@@ -10,7 +10,7 @@ const {
   Before,
   After,
   defineStep
-} = require("cypress-cucumber-preprocessor/lib/resolveStepDefinition");
+} = require("${__dirname}/resolveStepDefinition");
 const Given = (window.Given = window.given = given);
 const When = (window.When = window.when = when);
 const Then = (window.Then = window.then = then);
@@ -20,5 +20,5 @@ window.defineParameterType = defineParameterType;
 window.defineStep = defineStep;
 const {
   createTestsFromFeature
-} = require("cypress-cucumber-preprocessor/lib/createTestsFromFeature");
+} = require("${__dirname}/createTestsFromFeature");
 `;

